docs/howto/add_pagination_links.md

```diff
@@ -0,0 +1,46 @@
+# How to add pagination links
+
+Pagination links will be included in your response automatically as long as the resource is paginated and if you are using a ```JSON-API``` adapter. The others adapters does not have this feature.
+
```

Comment:
Up to you, but I think it could be linked from the CollectionSerializer above https://github.com/rails-api/active_model_serializers/pull/1041/files#diff-04c6e90faac2675aa89e2176d2eec7d8R121